test/lib/random_schema.cc

```diff
@@ -981,7 +980,7 @@ std::vector<data_model::mutation_description::key> random_schema::make_pkeys(siz
 
     uint32_t i{0};
     while (keys.size() < n) {
-        keys.emplace(_partitioner.decorate_key(*_schema, partition_key::from_exploded(make_partition_key(i, val_gen))));
+        keys.emplace(_schema->get_partitioner().decorate_key(*_schema, partition_key::from_exploded(make_partition_key(i, val_gen))));
```

Comment:
nitpick: I think at this point you already have dht::decorate_key() which does this a bit shorter.

Reply:
Thanks. Fixed.